j5/components/buzzer.py

```diff
@@ -0,0 +1,30 @@
+"""Classes for the Buzzer support."""
+
+from abc import ABCMeta, abstractmethod
+
+from j5.boards import Board
+from j5.components import Component
+
+
+class BuzzerInterface(metaclass=ABCMeta):
+    """An interface containing the methods required to control an buzzer."""
+
+    @abstractmethod
+    def buzz(self, board: Board, identifier: int, length: int,
+             frequency: int = None, note: str = None) -> None:
```

Comment:
Should we create a `Pitch` Type so that only one is needed. 
In the `Pitch` type, we should `Union` an `Enum` of note letters, and a int for the frequency.

If we change the name to `Piezo`, then a frequency is required.